test/unit/queues.test.ts

```diff
@@ -226,6 +226,36 @@ describe("queues", () => {
         });
     });
 
+    it("allows changing the max concurrency", async () => {
+        const { cfTemplate } = await runServerless({
```

Comment:
You can retrieve serverless current version much more easily :
```suggestion
        const { cfTemplate, serverless } = await runServerless({
```

and then, you have access to `serverless.version` that outputs the version of the framework :)